src/terminal/adapter/InteractDispatch.cpp

```diff
@@ -62,7 +62,7 @@ void InteractDispatch::WriteString(const std::wstring_view string)
 {
     if (!string.empty())
     {
-        const auto codepage = _api.GetConsoleOutputCP();
+        const auto codepage = _api.GetOutputCodePage();
         InputEventQueue keyEvents;

```

Comment:
I forgot to mention that I noticed we're using the output code page here to generate input events, which seemed wrong to me. Was that intentional?

That codepage is only used if the `UseNumpadEventsForClipboardInput` feature is enabled (which doesn't appear to be the case), so I don't think it's breaking anything at the moment, but it's still worth fixing if it's incorrect.

Reply:
That's an excellent question. Personally I'm not sure, though I suspect that it was an oversight.

`UseNumpadEvents...` was a last minute "don't break Windows conhost users right before a big release" feature flag, which we should almost certainly just remove now... :)